packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 43.13.0 - August 23, 2018
+
+### Added
+
+#### salesforcedx-vscode-apex
+
+- VS Code Apex Test Sidebar that allows you to view, run, and interact with Apex tests in your project. ([PR #552](https://github.com/forcedotcom/salesforcedx-vscode/pull/552))
```

Comment:
Should we also put a link to the new section that @ruthemmanuelle just wrote yesterday (thanks again for that Ruth)?

Reply:
Yeah, let's do that. After Emily reviews my PR I'll merge it and add a link here.